.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,15 @@
+# Automatically approve PRs made by Dependabot
+
+name: Auto Approve Dependabot PR's
+
+on:
+  pull_request
```

Comment:
Will this work only if all the **CI / build** steps for a PR were successful?

Reply:
This will give the "review approval" for dependabot PR's but those other checks will still need to run and be required to pass before merging becomes unblocked.

Reply:
Understood. Thanks.